test/cptree.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+import Turtle
+import Filesystem.Path.CurrentOS ()
+import System.IO.Temp (withSystemTempDirectory)
+import Control.Monad (unless)
+
+check :: String -> Bool-> IO ()
+check errorMessage successs = unless successs $ error errorMessage
+
+-- This test fails by hanging
```

Comment:
Sorry for this comment I start my test by copying pasting an other one and I forgot to remove the comment!